### DISCUSSION TITLE
Is it possible to update auth providers?

### DISCUSSION THREAD

**Discussion by galacticgibbon** [supabase/supabase#13887] [Questions]
I have google/apple/email auth in my app.

Currently, if someone signs in with either, is it possible to link/unlink various providers?

e.g. 
1. user signs up with email, later on adds google login, or 
2. user signs up with google, later wants to unlink google and just use email

**Comment by andrzejkupczyk** [comment]
Yes, you can [link](https://supabase.com/docs/reference/javascript/auth-linkidentity) or [unlink](https://supabase.com/docs/reference/javascript/auth-unlinkidentity) identities. However, it is important to note that this process may not work as expected when using an email provider.
